src/main/java/software/amazon/cloudformation/resource/ResourceTypeSchema.java

```diff
@@ -75,6 +76,11 @@ public ResourceTypeSchema(Schema schema) {
             : null;
         this.unprocessedProperties.remove("$schema");
 
+        this.unprocessedProperties.computeIfPresent("replacementStrategy", (k, v) -> {
+            ((ArrayList<?>) v).forEach(p -> this.replacementStrategy.add(p.toString()));
+            return null;
```

Comment:
are we not going to provide the default value here if they aren't present? ["create", "delete"]

Reply:
Ah I was thinking about handling in the workflows where if a schema does not have replacementStrategy info will set it to [create, delete]. But doing it here seems neater to me. 

Is there like a convention on how not present variables are handled?